apps/trading/components/rewards-container/rewards-history.tsx

```diff
@@ -258,6 +258,14 @@ export const RewardHistoryTable = ({
     return colDefs;
   }, []);
 
+  if (!pubKey) {
```

Comment:
Why is this added? This table should still be able to render without a connected wallet so you can see all rewards

Reply:
updated